### PR TITLE
fix: plugins workers without context

### PR DESCRIPTION
**Pull request by scottinet**
:warning: Must be merged after #316 :warning: 

Workers plugins don't have a plugin context provided to their `init` function, causing 2 problems:

- a discrepancy between worker plugins and every other plugins in their `init` method prototype
- the inhability for worker plugins to use a plugin context

This PR solves the issue by providing worker plugins with a properly instantiated plugin context. 
With one exception: there is no `accessors` in the provided context, since worker plugins run in a different process and are isolated from kuzzle.

### CODE REVIEW

File: docker-compose/debug.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ kuzzle:
   command: /run-debug.sh
   volumes:
     - "..:/var/app"
+    - "../kuzzle-enterprise-probe:/probes"
```
Comment (ballinette) [inline docker-compose/debug.yml]: something to do with workers plugin ?

**Comment by ballinette** [comment]
despite my question about docker-compose file, +1

**Comment by benoitvidis** [comment]
:+1:

**Comment by stafyniaksacha** [comment]
+1

**Pull request merged**